ExpressMapper.Tests NET40/CollectionTests.cs

```diff
@@ -31,6 +31,64 @@ public void AutoMemberMap()
         }
 
         [Test]
+        public void AutoMemberMap_ToNonGenericListInherited() {
+          Mapper.Register<TestModel, TestViewModel>();
+          Mapper.Register<Size, SizeViewModel>();
+          Mapper.Register<Country, CountryViewModel>();
+          Mapper.Compile();
+
+          var testData = Functional.CollectionAutoMemberMap();
+
+          var result = Mapper.Map<List<TestModel>, NonGenericCollectionInhertedFromList>( testData.Key );
+
+          Assert.AreEqual( result.Count, testData.Value.Count );
+
+          for ( var i = 0; i < result.Count; i++ ) {
+            Assert.AreEqual( result[i], testData.Value[i] );
+          }
+        }
+
+        [Test]
+        public void AutoMemberMap_ToLinkedList() {

```

Comment:
This didn´t fail before, but i thought it was useful as a test protecting against assuming use of IList.